### ISSUE TITLE
Prefabs: Unused prefab assets need to be stripped even If Game Option for stripping unused assets is switched OFF

### ISSUE THREAD

**Issue by rwkay** [YoYoGames/GameMaker-Bugs#8312]
### Description

(as title)

Also better reporting of what has been stripped out, something like

```
-------------------------------------------------------
NOTE: 167 Unused Assets found (and will be removed) -

GMAnimCurve :: animation_curve_bounce_a_lot, animation_curve_bounce_twice, animation_curve_one_way_end, 
GMFont :: fntHuge, Font5
GMObject :: oAreaLaser, oJumpSmash, oLaser, oPunch, oTest, oWaveManager
GMShader :: ShaderAlpha, ShaderHitFlash, ShaderRed, ShaderWhiteSpawn
GMSound :: sndHeal, sndMenuChange, sndMenuConfirm, sndNewWave
GMSprite :: sFaceDown, sFaceLeft, sFaceRight, sFaceUp, sKey0, sKey1, sKey2, sKey3, sKey4, sKey5, sKey6, sKey7, sKey8, sKey9, sKeyA, sKeyAlt, 
-------------------------------------------------------
```

### Expected Change

if user has disabled asset stripping we should still strip out filter / effects / particles and other things that they do NOT use 

### Steps To Reproduce

1. Start GameMaker
2. 
3. 
4. See the issue

### How reliably can you recreate this issue using your steps above?

Always

### Which version of GameMaker are you reporting this issue for?

2024.1100 (Betas)

### Which platform(s) are you seeing the problem on?

Windows

### Sample Package Attached?

- [x] I have attached my bug package

### Sample Project Added?

- [x] I have included a small sample project

**Comment by rwkay** [comment]
Feature added in 2024.11 - if remove unused assets has been switched off then the algorithm is still run  (so unused prefab assets are are still stripped) but all the main project assets are marked as being used.

We have also added the unused asset reporting so they are better presented - at least for some definition of better.

**Comment by Emc1923** [comment]
This doesn't seem to be working as expected as of  IDE v2024.1100.0.688  Runtime v2024.1100.0.709.

Output log example when the remove unused asset option is unselected : [outputlog_OFF.txt](https://github.com/user-attachments/files/17817239/outputlog_OFF.txt)
And when is it selected: [outputlog_ON.txt](https://github.com/user-attachments/files/17817248/outputlog_ON.txt)

**Comment by rwkay** [comment]
What do you feel is actually wrong? the output is exactly what I would expect

**Comment by Emc1923** [comment]
If unused Prefab assets are being stripped even when the option is off I would expect the warning and list of assets removed to appear in both cases on the output. The project I used to test only had Prefabs.

**Comment by rwkay** [comment]
Can you please attach the project then - as only the assets in the main project should be getting reported so I need to work out why they are getting reported...

**Comment by Emc1923** [comment]
https://api.gamemaker.io/api/github/downloads/f434be3f-3aad-43b5-9ce8-a66fdfc09f80

**Comment by rwkay** [comment]
Fixed in 2024.11

**Comment by Emc1923** [comment]
Verified fixed in  IDE v2024.1100.0.691  Runtime v2024.1100.0.711 - prefab assets are not reported.
